stories/Icons.stories.tsx

```diff
@@ -1,704 +1,21 @@
-      <Icon iconName="search"></Icon>
-      <br></br>
-      <span>search</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="send"></Icon>
-      <br></br>
-      <span>send</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="settings"></Icon>
-      <br></br>
-      <span>settings</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="share"></Icon>
-      <br></br>
-      <span>share</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="signals-1"></Icon>
-      <br></br>
-      <span>signals-1</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="signals"></Icon>
-      <br></br>
-      <span>signals</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="smileys-and-people"></Icon>
-      <br></br>
-      <span>smileys-and-people</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="spam"></Icon>
-      <br></br>
-      <span>spam</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="speaker-off"></Icon>
-      <br></br>
-      <span>speaker-off</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="speaker-on"></Icon>
-      <br></br>
-      <span>speaker-on</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="star"></Icon>
-      <br></br>
-      <span>star</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="symbols"></Icon>
-      <br></br>
-      <span>symbols</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="symphony"></Icon>
-      <br></br>
-      <span>symphony</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="table"></Icon>
-      <br></br>
-      <span>table</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="top"></Icon>
-      <br></br>
-      <span>top</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="travel-and-places"></Icon>
-      <br></br>
-      <span>travel-and-places</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="unread"></Icon>
-      <br></br>
-      <span>unread</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="up-arrow"></Icon>
-      <br></br>
-      <span>up-arrow</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="very-important"></Icon>
-      <br></br>
-      <span>very-important</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="video"></Icon>
-      <br></br>
-      <span>video</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="voice"></Icon>
-      <br></br>
-      <span>voice</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="world"></Icon>
-      <br></br>
-      <span>world</span>
-    </div>
-    <div className="preview">
-      <Icon iconName="zoom-out"></Icon>
-      <br></br>
-      <span>zoom-out</span>
-    </div>
+    {sortName.map((value, index) => {
```

Comment:
Nice! 👍 😃 
A minor remark, if you want the code smaller, you can replace
```
{sortName.map((value, index) => {return (<div>...</div>)})}
```
with 
```
{sortName.map((value, index) => (<div>...</div>))}
```

because `(expression)` will return the value of the expression 😉 .